core/src/test/java/org/apache/calcite/sql/test/SqlAdvisorTest.java

```diff
@@ -210,6 +210,8 @@ class SqlAdvisorTest extends SqlValidatorTestCase {
           "KEYWORD(NULLIF)",
           "KEYWORD(OCTET_LENGTH)",
           "KEYWORD(OVERLAY)",
+          "KEYWORD(PERCENTILE_CONT)",
+          "KEYWORD(PERCENTILE_DISC)",
```

Comment:
Why do we need to add them here? Is there any tests that could verify this? I'm asking because I see you added other keywords to `SqlParserTest`, but only add these two here.

Reply:
Same behavior as the question above. CALCITE-5564 (https://github.com/apache/calcite/pull/3210) introduced PERCENTILE_CONT & PERCENTILE_DISC as functions. It delete these 2 keywords in SqlAdvisorTest to let the testcases passed, if we reset it as ReservedKeywords, we need to reset it either. Otherwise, [CALCITE-5564](https://issues.apache.org/jira/browse/CALCITE-5564) and SqlAdvisorTest will be failed.